stubs/chevron/chevron/main.pyi

```diff
@@ -2,6 +2,8 @@ from _typeshed import StrOrBytesPath
 from typing import Any
 from typing_extensions import TypeAlias
 
+version: str
+
```

Comment:
I'm going to be picky here: I think it would be better if this was allowlisted. The fact that this constant is imported from `.metadata` in this file is an implementation detail; if users are accessing the constant, they should import it from `chevron.metadata` instead of importing it from `chevron.main`.

Reply:
There's something strange. Why is `version` reported and `render` is not? They are imported together: https://github.com/noahmorrison/chevron/blob/5e1c12827b7fc3db30cb3b24cae9a7ee3092822b/chevron/main.py#L7-L8

Reply:
stubtest is smart enough to know that `render` comes from another module because it's a function. But unfortunately, `version` is just a str, so Python doesn't track which module it comes from.